html/theme.css

```diff
@@ -0,0 +1,88 @@
+
+
+
+/* I tried to mimic VS Code's Dark+ theme with these colors: */
+
+.hljs-literal,
+.hljs-keyword {
```

Comment:
These hard-coded colors might not appear clearly in light themes. For example, with Solarized Light:

<img width="242" alt="image" src="https://user-images.githubusercontent.com/4662568/99601845-3a086d80-2a3b-11eb-9da8-69110df9e3cc.png">

Maybe we need the following to make the color work under all types of themes?

```css
body.vscode-light .hljs-literal,
body.vscode-light .hljs-keyword {
    color: ???
}

body.vscode-dark .hljs-literal,
body.vscode-dark .hljs-keyword {
    color: ???
}

body.vscode-high-contrast .hljs-literal,
body.vscode-high-contrast .hljs-keyword {
    color: ???
}
```

Reply:
The best way to solve this would be to access the colors of the current theme, either directly as RGB values or through CSS variables (as is possible for [theme colors](https://code.visualstudio.com/api/references/theme-color)). AFAIK, this is currently not possible, but comes up frequently ([e.g. here](https://github.com/microsoft/vscode/issues/32813)).

A somewhat hacky workaround would be to manually adapt the CSS files from the default themes and select the correct one using the approach you described (though we would still be limited to dark/light/hc). Taking this one step further, we could also try to automatically parse/convert the CSS files and maybe even let users pass their own theme files (for people who use themes that are vastly different from the default ones).

Either way, a clean solution would either require changes to the colors exposed by vscode, or take some time to understand and convert the involved CSS, which is why I'd consider this a "long-term todo".

Reply:
I think the current syntax highlighting works ok with the three default themes. To make sure the help view is still usable by people with very different themes, I'd propse adding an option to disable syntax highlighting altogether

Reply:
I think you are right. The current syntax highlighting already works quite well for most cases.